samza-core/src/main/scala/org/apache/samza/util/Util.scala

```diff
@@ -81,10 +81,17 @@ object Util extends Logging {
    * Instantiate a class instance from a given className.
    */
   def getObj[T](className: String) = {
-    Class
-      .forName(className)
-      .newInstance
-      .asInstanceOf[T]
+    try {
+      Class
+        .forName(className)
+        .newInstance
+        .asInstanceOf[T]
+    } catch {
+      case e: Throwable => {
```

Comment:
Is it better to handle specific exceptions here, rather than `Throwables` ?

Reply:
Normally I would catch specific exceptions if the exception handling was more extensive, but the intent is just to log the name of the class (which is currently not reported in the logs) that caused the error before a throw. On the other hand, specific exceptions could be caught with slightly different error messages, but I'm not sure what other information could be logged that's not already conveyed in the stack trace reported from the exception. Thoughts? Thanks for reviewing!